src/IntegrationWrapperV2.py

```diff
@@ -52,8 +57,9 @@
 args = parser.parse_args()
 
 #Check if file (Input.csv) is in csv
+print(args.file)
 if not os.path.isfile(args.file): raise Exception ("File does not exist")
-refDir = {"bdel":config["referenceFolderBasinDel"],"bchar":config["referenceFolderBasinChar"]}
+refDir = {"bdel":Shared.CheckDirectoryExists(config["referenceFolderBasinDel"]),"bchar":Shared.CheckDirectoryExists(config["referenceFolderBasinChar"]), "flowstats":Shared.CheckDirectoryExists(config["referenceFolderFlowStats"])}
```

Comment:
I am getting errors at this line "AttributeError: module 'WIMLib.Shared' has no attribute 'CheckDirectoryExists'" and I am little confused were this `CheckDirectoryExists` comes from.

There is code for the `Shared.GetWorkspaceDirectory` https://github.com/jknewson/WIMLib/search?q=GetWorkSpaceDirectory, and `Shared.readCSVFile` https://github.com/jknewson/WIMLib/search?q=readCSVFile, but not for `Shared.CheckDirectoryExists` https://github.com/jknewson/WIMLib/search?q=CheckDirectoryExists.

Reply:
Oh! Haha I edited the Shared.py locally and didn't register it was an external file. 😅 I'm too distracted today